src/mudlet-lua/lua/Other.lua

```diff
@@ -1045,8 +1040,64 @@ function packageDrop(event, fileName, suffix)
     -- Cyan and Orange; see cTelnet::postMessage for color comparison
   end
 end
+
+
+--- Installs package from url
+-- @param url
+function installPackageFromUrl(url)
+  local fileName, suffix = url:gmatch("([^/]+)%.([^.]+)$")()
+  if suffix and not table.contains(acceptableSuffix, suffix) then
+    return
+  end
+
+  local destination = string.format("%s/%s.%s", lfs.currentdir(), fileName, suffix)
```

Comment:
Not safe to use binary location as the destination - it's not writeable on all platforms, for example this doesn't work on macOS (with an amazing error message /s):

![image](https://user-images.githubusercontent.com/110988/106377556-14360800-639e-11eb-9e29-d93ed98afc18.png)

I'd write it into the profile folder instead.